frictionless/project/database.py

```diff
@@ -201,15 +201,21 @@ def list_records(self) -> List[IRecordItem]:
                     self.records.c.type,
                     self.records.c.updated,
                     self.records.c.tableName,
+                    self.records.c.report,
                 )
             )
             items: List[IRecordItem] = []
             for row in result:
+                errorCount = 0
+                if row.report:
```

Comment:
Can you please add a TODO that it's very ineffective and we need to add `errorCount` as a database column in the future?

Reply:
Added here https://github.com/frictionlessdata/application/issues/146